### DISCUSSION TITLE
I am unable to access documents under the self built collection

### DISCUSSION THREAD

**Discussion by aiqubits** [avitorio/outstatic#167] [Q&A] [answered]
site: https://welljobcns.vercel.app/,
github repository: https://github.com/aiqubits/welljobcns

I created collections and documents on Vercel, and the Vercel app displayed successful deployment. The GitHub repository also saw the correct file path, but when accessing the self built document, it still reported that the page could not be found, such as accessing https://welljobcns.vercel.app/jobs/firstjob

**Comment by aiqubits** [comment]
![image](https://github.com/avitorio/outstatic/assets/35585232/c6fce564-fef6-45c2-a7d3-305259a9d6b4)

**Reply by avitorio** [reply, marked as answer]
You need to create the correct route and pages to fetch the data for the collection.

You can check the code from your repository for the posts and projects collections and create a route for jobs.
